Setup/Patch/Data/ImageRolesNotFilterable.php

```diff
@@ -0,0 +1,65 @@
+<?php
+
+namespace Macopedia\Allegro\Setup\Patch\Data;
+
+use Magento\Eav\Setup\EavSetup;
+use Magento\Eav\Setup\EavSetupFactory;
+use Magento\Framework\Setup\ModuleDataSetupInterface;
+use Magento\Framework\Setup\Patch\DataPatchInterface;
+
+/**
+ * Image role patch script
+ */
+class ImageRolesNotFilterable implements DataPatchInterface
+{
+    /** @var EavSetupFactory */
+    private $eavSetupFactory;
+
+    /** @var ModuleDataSetupInterface */
+    private $moduleDataSetup;
+
+    /**
+     * ImageRoles constructor.
+     * @param EavSetupFactory $eavSetupFactory
+     * @param ModuleDataSetupInterface $moduleDataSetup
+     */
+    public function __construct(
+        EavSetupFactory $eavSetupFactory,
+        ModuleDataSetupInterface $moduleDataSetup
+    ) {
+        $this->moduleDataSetup = $moduleDataSetup;
+        $this->eavSetupFactory = $eavSetupFactory;
+    }
+
+    /**
+     * @return array|string[]
+     */
+    public static function getDependencies()
+    {
+        return [];
```

Comment:
Missing dependency to patch `ImageRoles` so `updateAttribute` method may be executed before `createAttribute` method for new installations.

Reply:
thx, done